src/sidebar/util/test/groups-test.js

```diff
@@ -102,14 +102,14 @@ describe('sidebar.util.groups', () => {
       {
         description: 'sets `isScopedToUri` to true if `scopes` is missing',
         scopes: undefined,
-        shouldBeSelectable: true,
+        isScopedToUri: true,
         uri: 'https://foo.com/bar',
       },
       {
         description:
-          'sets `isScopedToUri` to true if `scopes.enforced` is false',
-        scopes: { enforced: false },
-        shouldBeSelectable: true,
+          'sets `isScopedToUri` to false if `scopes.enforced` is false and patterns do not match',
+        scopes: { enforced: false, uri_patterns: ['http://bar.com*'] },
```

Comment:
Is this test still relevant now that `isScopedToUri`'s return value no longer depends on the `enforced` property?

Reply:
Originally I left it to make sure the change I made was behaving correctly but yes, since it isn't dependent on enforced anymore this isn't necessary.